src/test/java/com/training/web/tests/flipkart/FlipkartTest.java

```diff
@@ -104,5 +120,6 @@ public void testCartAddition() throws ParseException {
         }
         softAssertions.assertThat(totalCartPrice).as("Price doesn't match")
                 .isGreaterThanOrEqualTo(totalPrice);
+        getWebDriver().close();
```

Comment:
Why do we have this in test? shouldn't this be in after each in the base test - since this is a common code?